### DISCUSSION TITLE
SQLite - createMany() availability

### DISCUSSION THREAD

**Discussion by tajnymag** [prisma/prisma#7372] [Q&A] [answered]
Hi, I was wondering, what was the reason for disallowing createMany on sqlite datasources. I've tried searching the issues and blog posts, but haven't found any in-depth explanation. What makes sqlite incapable of performing this method?

createMany() is a really useful feature and I believe it should be implementable with sqlite's native INSERT INTO support. 

[the pull request I'm talking about](https://github.com/prisma/prisma/pull/5389)

-----------

In a similar vein, but a different topic, sqlite should be able to support JSON querying, thanks to the JSON1 extension, thus allowing a greater feature parity among other engines.

**Comment by ryands17** [comment, marked as answer]
@Tajnymag 👋 

We have a request for Json support here that you can add a 👍 and follow: https://github.com/prisma/prisma/issues/3786

As for `createMany` with `insert into` support, you can create a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) so that we can look into this.

**Reply by tajnymag** [reply]
Thanks. I'm sorry for not noticing the existing JSON support issue.

I'll create a feature request for `createMany`  :+1:

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
